pkg/agent/controller/traceflow/packetin.go

```diff
@@ -131,8 +131,13 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*opsv1alpha1.Tracefl
 		}
 		ob := new(opsv1alpha1.Observation)
 		ob.Component = opsv1alpha1.NetworkPolicy
-		ob.ComponentInfo = openflow.GetFlowTableName(openflow.EgressRuleTable)
-		ob.Action = opsv1alpha1.Forwarded
+		if tableID == uint8(openflow.EgressMetricTable) {
```

Comment:
Sorry, but why it is EgressMetricTable here? Probably you should add some comments to explain.

Reply:
If user added ANP with "drop" action, the drop rule will be at EgressMetricTable/IngressMetricTable.
We also mirror metricFlows if the action is "drop" and set it to "controller", to send the dropped packet to Antrea agent, and the packet will be processed here.